### DISCUSSION TITLE
Inter-discussion linking

### DISCUSSION THREAD

**Discussion by aredridel** [community/community#3469] [Discussions]
I would find it useful to see links between discussions (as links between issues and PRs do), where there is a (low emphasis) connection from the linked-to discussion back to where it's mentioned.

**Comment by Stvad** [comment]
seems like a duplicate of https://github.com/github/feedback/discussions/2971

**Reply by aredridel** [reply]
Similar, but between discussions, rather than between issues and discussions! Definitely worth merging.

**Reply by aredridel** [reply]
.... if this feature existed that one wouldn't need an update to include this :D

**Comment by evi-liu** [comment]
Thanks for the feedback! What are the use cases where it would be most valuable to link between Discussions?

**Reply by aredridel** [reply]
"We talked about that [here]", or "this information is also useful for the people talking about [that thing]" are both things I would do heavily. Especially in a company context where old projects get picked back up, and requirements change and one of the big challenges is keeping people working on different sub-projects aligned, the casual mentioning across threads would be useful for ambient context-keeping.
